### DISCUSSION TITLE
DAI 2023-2024 - Practical work 1 - Alan Sottile and Calvin Graf

### DISCUSSION THREAD

**Discussion by CalvinGraf1** [heig-vd-dai-course/heig-vd-dai-course#61] [Show and tell]
Groupe pour le labo 1

**Comment by hadrylouis** [comment]
Could you please tell us a bit about what you plan to do for this practical work? :)

**Reply by CalvinGraf1** [reply]
Bien sûr !

Notre idée pour le moment est d'inverser les minuscule en majuscule et vice-versa. Et une fois que nous avons réussi à implémenter notre fonctionnalité et que tout semble fonctionner, d'ajouter plusieurs autres fonctionnalités selon le temps que nous avons, comme par exemple :

- Inverser le sens du texte (Hello World → World Hello)
- Faire une "symétrie" de l'alphabet (a → z, b → y, c → x, etc.)
- etc.

C'est une liste non exhaustive, nous avons déjà pensé à ces 2 fonctions mais nous en ajouterons d'autres le moment venu.
